tests/cli/conftest.py

```diff
@@ -55,3 +71,34 @@ def do_run(*args):
         return testdir.run(*args)
 
     return do_run
+
+
+def _set_up_initial_project(testdir: Testdir):
+    # copy initial project files
+    testdir.copy_example("rasa/cli/initial_project/actions.py")
```

Comment:
why not use our scaffolding function that already does the copying?

Reply:
see comment below